daemon/graphdriver/driver_unsupported.go

```diff
@@ -1,4 +1,4 @@
-// +build !linux
+// +build !linux,!windows
```

Comment:
I believe you need to have FsNames/Priority for this to compile on unsupported platforms. (I'm not sure why this file exists in the first place)